src/client/containers/BoardPage/BoardPage.container.js

```diff
@@ -1,3 +1,75 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './BoardPage.style.css';
+import OwnerBoardPage from './OwnerBoardPage';
+import MemberBoardPage from './MemberBoardPage';
+import { useBoard } from './BoardProvider';
+import { useUser } from '../../firebase/UserContext';
+import { useParams } from 'react-router-dom';
 
-export const Board = () => <div className="Board ">The Board!</div>;
+export default function BoardPage() {
+  const [errorCode, setErrorCode] = useState(null);
+  const {
+    isBoardLoading,
+    setBoardLoading,
+    boardInfo,
+    setBoardInfo,
+  } = useBoard();
+  const { boardId } = useParams();
+  const { user } = useUser();
+  const userId = user[0].id;
+  useEffect(() => {
+    const fetchingBoardApi = async () => {
+      const API_URL = `/api/users/${userId}/boards/${boardId}`;
+      try {
+        const apiResponse = await fetch(API_URL);
+        if (apiResponse.ok) {
+          const apiData = await apiResponse.json();
+          apiData[0].hasPassedDeadline = function () {
+            const deadlineDate = new Date(this.deadline);
+            const today = new Date();
+            return today > deadlineDate;
+          };
+          setBoardInfo(apiData[0]);
+          setBoardLoading(false);
```

Comment:
Still using the context state here.
See here:
https://github.com/HackYourFuture-CPH/simply-name.it/pull/233#issuecomment-907803223

Reply:
changed it in this pr https://github.com/HackYourFuture-CPH/simply-name.it/pull/254